docker-compose.staging.yml

```diff
@@ -11,7 +11,7 @@ services:
   reverse_proxy:
     image: "nginx:1.19.5"
     volumes:
-      - ./docker/reverse_proxy/www:/usr/share/nginx/html
+      - ./mapbox/styles:/usr/share/nginx/html
```

Comment:
```suggestion
      - ./mapbox/styles:/usr/share/nginx/html
```

I would avoid the name mapbox, maybe map-styles would be better.

Reply:
I was sure someone would dislike this, I also do 😆 Hm, I find map-styles does not fit to martin. Maybe map-tiles?

Reply:
Done